src/ProgressOnderwijsUtils/ProgressOnderwijsUtils.csproj

```diff
@@ -10,6 +10,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="AngleSharp" Version="0.14.0" />
+    <PackageReference Include="DotNetZip" Version="1.14.0" />

```

Comment:
waar komt deze vandaan?

Reply:
Achtergrond is deze issue: utils -> ZlibWithDictionary -> DotNetZip -> System.Security.Permissions <--version conflict.

toen heb ik de hele keten maar eens gebumpt, en dat werkt.

...maar daarvoor blijkt de DotNetZip niet uit te maken. (sowieso zijn al die versions gewoon compatibel ondanks semver major version).


Als je'm liever niet bumpt (op zich wel mee eens), dan:

```suggestion
```